src/main/java/spark/Base64.java

```diff
@@ -16,16 +16,13 @@
  */
 package spark;
 
-import java.io.IOException;
-
-import sun.misc.BASE64Decoder;
-import sun.misc.BASE64Encoder;
-
-
 class Base64 {
 
     //CS304 Issue link:https://github.com/perwendel/spark/issues/1061
 
+    private static final java.util.Base64.Encoder urlEncoder = java.util.Base64.getUrlEncoder();
```

Comment:
I don't know if this is valid for Java 8s Base64 class but I have a vague memory that it's faster not assigning an encoder/decoder to a static instance and then re-use for each decoding/encoding.

Reply:
maybe it had to do with not being threadsafe...hmmm. Do you know? I'll try to find out.